modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -541,7 +541,14 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     return scrollContainer;
   };
 
-  const scrollIntoView = (elm: HTMLElement, alignToTop?: boolean) => ScrollIntoView.scrollElementIntoView(editor, elm, alignToTop);
+  const scrollIntoView = (elm?: HTMLElement, alignToTop?: boolean) => {
+    if (Type.isNonNullable(elm)) {
+      ScrollIntoView.scrollElementIntoView(editor, elm, alignToTop);
+    } else {
+      ScrollIntoView.scrollRangeIntoView(editor, getRng(), alignToTop);
```

Comment:
```suggestion
      ScrollIntoView.scrollRangeIntoView(editor, getRng());
```
`alignToTop` can't be defined here if we've taken this code path. As such, we should either omit it from the arguments, explicitly pass in false, or write a proper overload where you could call something like `scrollIntoView(alignToTop)` on its own

Reply:
You're incorrect here sorry Neil, you can quite happily pass `scrollIntoView(undefined, false)`. So this is not something we should be doing...